### DISCUSSION TITLE
How to use monai.losses.TverskyLoss

### DISCUSSION THREAD

**Discussion by Guoxt** [Project-MONAI/MONAI#6680] [Q&A]
A 2D segmentation model with a network output of 2-channel logical values without softmax processing. For example, the network output dimension is (B, 2,128,128). And the dimension of the target is (B, 1,128,128). How to set the parameters of the Loss function in this case?

**Comment by wyli** [comment]
thanks, the answer would depend on the task, possibly please consider `to_onehot_y=True`, there are other viable shape and parameter combinations in the test cases: https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_tversky_loss.py#L85

**Reply by Guoxt** [reply]
Thank you for your answer. Another question is, is this Loss function also applicable to multi category tasks?
